csuntechlab/iSTART
# STR-142 -- FE: Multi-Question Multiple-Choice Survey Slide + Survey Feedback Slide Template

# Description
Creates templates and simple functionality for the components mentioned in PR
  
# Testing
1. log in as april 
2. click on a modal
3. Look at multiple choice survey slide and make sure it looks ok in every viewport
4. click on a choice and you will receive visual feedback, the continue button will also now be of darker color
5. on ``module.vue`` change the contentType property from ``multiChoiceSurvey`` to ``multiChoiceSurveyResults``
6. make sure that looks ok too
7. initially, button will be disabled, it will be enabled after 5 seconds 

# Installation
'Are there any dependencies the reviewer needs to download?'
